mlrun/datastore/targets.py

```diff
@@ -1249,7 +1252,31 @@ def get_spark_options(self, key_column=None, timestamp_key=None, overwrite=True)
         return spark_options
 
     def purge(self):
-        pass
+        import snowflake.connector
```

Comment:
I think it needs to be added to `extra-requirements.txt` and `dependencies.py`.

Reply:
you are right, moving it from dev-requirements